apps/desktop/desktop/src/main/resources/css/chat.css

```
@@ -153,6 +153,69 @@
     -fx-background-color: -bisq-dark-grey-50 !important;
 }
 
+.chat-markdown-message {
+    -fx-spacing: 3;
```

Biome (2.4.4)

[error] Unknown property is not allowed.

(lint/correctness/noUnknownProperty)